### DISCUSSION TITLE
Table - Filter Column by Numeric Expression

### DISCUSSION THREAD

**Discussion by Trovalo** [grafana/grafana#49894] [Dashboards, Panels & Visualizations]
The use case is pretty simple, being able to filter numeric columns by expression ie: `<10`, `>=60`, `>10<20`. (I took inspiration from BI tools for this syntax, but any will do)

This will be useful for all the cases in which you want a filter on a numeric column, to include/exclude a certain range of values.
As of now the only thing that helps is the column sorting.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
